crates/devolutions-pedm-shared/devolutions-pedm-client-http/src/apis/default_api.rs

```diff
@@ -17,8 +17,7 @@ use std::rc::Rc;
 use futures::Future;
 use hyper;
 
-use super::request as __internal_request;
```

Comment:
Changes in `devolutions-pedm-shared`/`terminal-streamer`/`video-streamer` were caused by `cargo +nightly fmt`